march_hardware_interface/include/march_hardware_interface/march_hardware_interface.h

```diff
@@ -13,6 +13,7 @@
 #include <ros/ros.h>
 #include <march_hardware_interface/march_hardware.h>
 #include <march_hardware_interface/march_temperature_sensor_interface.h>
+#include <march_hardware_interface/march_pdb_state_interface.h>
```

Comment:
Shouldn't be needed as it is included via the `march_hardware.h` in line 14, same goes for the `march_temperature_sensor_interface.h`